mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go.erb

```diff
@@ -2675,6 +2686,9 @@ func resourceBigQueryTableImport(d *schema.ResourceData, meta interface{}) ([]*s
 	if err := d.Set("deletion_protection", true); err != nil {
 		return nil, fmt.Errorf("Error setting deletion_protection: %s", err)
 	}
+	if err := d.Set("allow_resource_tags_on_deletion", false); err != nil {
+		return nil, fmt.Errorf("Error setting allow_resource_tags_on_deletion: %s", err)
+	}
```

Comment:
I think we may also need to mark it as beta only.

Reply:
Added